### DISCUSSION TITLE
"on_submit" for buttons, not just "on_click"

### DISCUSSION THREAD

**Discussion by tobsome** [flet-dev/flet#3556] [Ideas]
Hey guys,

thanks a lt for your work on Flet!

Using Flet now more and more I realized I'm missing the `on_submit` feature on buttons a lot. For a `TextField` it works fine but I would love to have the same feature to press a button when I press the Return key as well. 

What do you think?

**Comment by ndonkoHenri** [comment]
What you could alternatively do is to call the `on_click` handler of your button, when the user "submits" from the last `TextField`.
```py

import flet as ft


def main(page: ft.Page):
    submit_button = ft.ElevatedButton(
        "Submit", on_click=lambda _: print("Submitting ...")
    )

    def handle_submit(e):
        submit_button.on_click(None)

    page.add(
        ft.TextField(
            on_submit=handle_submit,
        ),
        submit_button,
    )


ft.app(main)
```
